src/Dapr.AzureFunctions.Extension/Triggers/DaprTopicTriggerAttribute.cs

```diff
@@ -16,8 +16,27 @@ namespace Dapr.AzureFunctions.Extension
     public class DaprTopicTriggerAttribute : Attribute
     {
         /// <summary>
-        /// Gets or sets the name of the topic.
+        /// Initializes a new instance of the <see cref="DaprTopicTriggerAttribute"/> class.
+        /// </summary>
+        /// <param name="name">The pub/sub name.</param>
+        public DaprTopicTriggerAttribute(string name)
+        {
+            this.Name = name;
+        }
+
+        /// <summary>
+        /// Gets or sets the pub/sub name.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the topic.
         /// </summary>
```

Comment:
Should topic be nullable?

Reply:
It's nullable because functions will infer the topic name from the method name. 

By default you only have to specify the pub/sub name, and then `method name -> topic name -> route`

See all of these tests: https://github.com/dapr/azure-functions-extension/pull/57/files#diff-aacd1a7dc891022469c3af44430dd0aeR98